### ISSUE TITLE
 shebang missing in python scripts

### ISSUE THREAD

**Issue by keesj** [vscode-restructuredtext/vscode-restructuredtext#182]
**Environment**
- [ ] Extension version: 116.0.0
- [ ] Visual Studio Code version: 1.39.2
- [ ] Operating System version: Linux Ubuntu 19.04
- [ ] Python version:Python 3.7.3
- [ ] sphinx version: 2.2.0
- [ ] A sample of conf.py or its important fragments:

**Expected behavior**
The extension should be able to start and show a preview when code starts

**Actual behavior**
My mouse shape is changed to a corss and system hangs until the mouse is pressed a few times. After that a message is displayed that code was not able to render the preview.

**Detailed Steps to reproduce the problem**
On  my setup I can open .rst files and click to view the preview (this is working) but if I have the preview open and I shutdown code the preview will no longer be visible when I start code again.

On Linux the file extention does not matter(much) for determining how to run a file and the scripts that are executed from a shell should start with a shebang e.g. [preview.py](https://github.com/vscode-restructuredtext/vscode-restructuredtext/blob/master/python/preview.py) should start with `#!/usr/bin/env python3` as can be read on [stack overflow](https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take) else the script is evaluated as a shell script. the first "message" is then 

![e](https://user-images.githubusercontent.com/57112/69147307-f3073580-0ad1-11ea-99f3-6f24d865bf97.png)

because "from" is a linux command to send mail. After that the [Image magick import](https://imagemagick.org/script/import.php) is called (and this shows the cross to select the window to import. after clicking a file called sys is created...)

The fix is trivial

```
From d1eb655167ceaa2b259b2648dac840d6a3af4901 Mon Sep 17 00:00:00 2001
From: Kees Jongenburger <kees.jongenburger@gmail.com>
Date: Tue, 19 Nov 2019 13:54:49 +0100
Subject: [PATCH] Add python3 shebang to execute preview.py as python script.

---
 python/preview.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/python/preview.py b/python/preview.py
index 9f78bb8..ccc32a5 100755
--- a/python/preview.py
+++ b/python/preview.py
@@ -1,3 +1,4 @@
+#!/usr/bin/env python3
 from __future__ import print_function
 import codecs
 import os.path
-- 
2.20.1
```

`

**Comment by lextm** [comment]
Hi, can you send a pull request?

**Comment by lextm** [comment]
Closed via #183
